app/src/main/java/org/simple/clinic/summary/RelativeTimestampGenerator.kt

```diff
@@ -63,3 +68,13 @@ data class OlderThanSixMonths(private val time: Instant) : RelativeTimestamp() {
     val timestampFormatter = DateTimeFormatter.ofPattern("d MMM, yyyy", Locale.ENGLISH)!!
   }
 }
+
+data class Future(private val time: Instant) : RelativeTimestamp() {
+  override fun displayText(context: Context): String {
+    return timestampFormatter.format(time.atZone(ZoneOffset.UTC).toLocalDateTime())
+  }
+
+  companion object {
+    val timestampFormatter = DateTimeFormatter.ofPattern("d MMM, yyyy", Locale.ENGLISH)!!
```

Comment:
Can you inject this via the `DateFormatterModule` instead of having a static instance?

Reply:
Never mind, I see the rest of the classes in this file also use the same convention. We'll anyway have to unify the relative date presentation across the codebase later, so let's leave this as is.